modules/products/tomcat/src/main/kotlin/com/metalbear/mirrord/products/tomcat/TomcatExecutionListener.kt

```diff
@@ -0,0 +1,93 @@
+@file:Suppress("UnstableApiUsage")
+
+package com.metalbear.mirrord.products.tomcat
+
+import com.intellij.execution.ExecutionListener
+import com.intellij.execution.process.ProcessHandler
+import com.intellij.execution.runners.ExecutionEnvironment
+import com.intellij.execution.target.createEnvironmentRequest
+import com.intellij.execution.util.EnvironmentVariable
+import com.intellij.execution.wsl.target.WslTargetEnvironmentRequest
+import com.intellij.javaee.appServers.run.configuration.RunnerSpecificLocalConfigurationBit
+import com.intellij.notification.NotificationType
+import com.intellij.openapi.components.service
+import com.metalbear.mirrord.CONFIG_ENV_NAME
+import com.metalbear.mirrord.MirrordLogger
+import com.metalbear.mirrord.MirrordProjectService
+import java.util.concurrent.ConcurrentHashMap
+
+private const val DEFAULT_TOMCAT_SERVER_PORT: String = "8005"
```

Comment:
Where did you get this from?

Reply:
From what I've seen online, looks like it's a default shutdown port for Tomcat.